### ISSUE TITLE
Automatically show graph/results + usernames

### ISSUE THREAD

**Issue by mkanwal** [mitar/council-app#23]
I know we initially wanted to create discussion independent of names, and just based on content.
And I know we wanted to have voting based on your opinion only, not to be biased based on current vote.

But it's killing me... I just keep opening the graph/username anyway to see.  We should just have these auto-displayed.

**Comment by mkanwal** [comment]
Perhaps graph is auto-displayed only after you switch your vote from Nothing to Abstain/Slider

**Comment by mitar** [comment]
You stalker! :-)

**Comment by mitar** [comment]
Yes, maybe this is interesting. How we want to know who wrote it. Maybe an opportunity for self-reflection. Should it really matter?

**Comment by mkanwal** [comment]
I think we all change the way we address each other based on who we are talking to ;)
Human nature...

**Comment by mitar** [comment]
Yea, so let's give opportunity to try something new. :-)
